src/AblyRest.php

```diff
@@ -1,7 +1,8 @@
 <?php
+namespace Ably;
 
-require_once dirname(__FILE__) . '/AuthMethod.php';
-require_once dirname(__FILE__) . '/Channel.php';
+use Ably\Exceptions\AblyException;
+use Ably\Exceptions\AblyRequestException;
```

Comment:
@kouno yep. It's the default behavior for Composer projects and pretty much every modern library or framework is using Composer.